src/main/java/io/vertx/kafka/client/consumer/impl/KafkaConsumerImpl.java

```diff
@@ -414,6 +415,91 @@ public void position(TopicPartition partition, Handler<AsyncResult<Long>> handle
   }
 
   @Override
+  public void offsetsForTimes(TopicPartition topicPartition, Long timestamp, Handler<AsyncResult<OffsetAndTimestamp>> handler) {
+    Map<TopicPartition, Long> input = new HashMap<>();
+    input.put(topicPartition, timestamp);
+
+    this.stream.offsetsForTimes(Helper.toTopicPartitionTimes(input), done -> {
+      if(done.succeeded()) {
+        // We know that this will result in exactly one iteration
+        for(org.apache.kafka.clients.consumer.OffsetAndTimestamp offsetAndTimestamp : done.result().values()) {
+          OffsetAndTimestamp r = new OffsetAndTimestamp(offsetAndTimestamp.offset(), offsetAndTimestamp.timestamp());
```

Comment:
@ppatierno: Thanks for the feedback! Tried to save a few keystrokes ;)

Renamed this var and a few others, too.